pkg/daemon/ceph/osd/volume.go

```diff
@@ -1162,10 +1162,6 @@ func callCephVolume(context *clusterd.Context, requiresCombinedOutput bool, args
 
 	// Do not use combined output for "list" calls, otherwise we will get stderr is the output and this will break the json unmarshall
 	f := context.Executor.ExecuteCommandWithOutput
-	if requiresCombinedOutput {
```

Comment:
Should we also remove `requiresCombinedOutput` from the signature too?

Reply:
right, removed.